Formula/ripgrep.rb

```diff
@@ -1,8 +1,8 @@
 class Ripgrep < Formula
   desc "Search tool like grep and The Silver Searcher"
   homepage "https://github.com/BurntSushi/ripgrep"
-  url "https://github.com/BurntSushi/ripgrep/archive/11.0.1.tar.gz"
-  sha256 "ba106404342160a66f703b8c9db9d45117c1a3664a0996822496bcb9f238b184"
+  url "https://github.com/BurntSushi/ripgrep/archive/11.0.2.tar.gz"
+  sha256 "b218bb70b0546bfb4aa94a5206dc528e96f194eb7b5c5de8e7db23e852788c35"
```

Comment:
```suggestion
  sha256 "0983861279936ada8bc7a6d5d663d590ad34eb44a44c75c2d6ccd0ab33490055"
```
Whoops, I supplied the zipped `sha256` instead of tar.gz 🤦‍♂ This should be correct.

Reply:
😆 no worries, just another commit 👍